src/libs/ajax/User.ts

```diff
@@ -76,15 +76,23 @@ export interface SetTerraUserProfileRequest {
 
 export const makeSetUserProfileRequest = (terraUserProfile: TerraUserProfile): SetTerraUserProfileRequest => {
   return {
-    firstName: terraUserProfile.firstName ?? 'N/A',
-    lastName: terraUserProfile.lastName ?? 'N/A',
-    title: terraUserProfile.title ?? 'N/A',
-    institute: terraUserProfile.institute ?? 'N/A',
+    // first name and last name are REQUIRED for this request, but should not ever be "N/A"
+    // TODO: determine if this request should fail if firstname or lastname is undefined
+    firstName: terraUserProfile.firstName! ?? 'N/A',
+    lastName: terraUserProfile.lastName! ?? 'N/A',
```

Comment:
The non-null assertion suggests that these are known to be defined, but the nullish coalescing operator suggests that they may be undefined. Which is true?

Suggest either
```suggestion
    firstName: terraUserProfile.firstName!,
    lastName: terraUserProfile.lastName!,
```

or
```suggestion
    firstName: terraUserProfile.firstName ?? 'N/A',
    lastName: terraUserProfile.lastName ?? 'N/A',
```

Reply:
I can now confirm that these are required parts of the `OidcTokenClaims`, so these should always be not be undefined if the user authenticated